doc/radosgw/adminops.rst

```diff
@@ -13,6 +13,9 @@ Get Usage
 
 Request bandwidth usage information.
 
+Note that you need add ``rgw enable usage log = true`` in [client.rgw] section
+of ceph.conf and restart the radosgw service. 
+
```

Comment:
@hrchu I'd modify the description a bit. First, the section name in the `ceph.conf` file might be different for different installations. I don't think this is the place to mention it (it's a reference, not  general setup instructions). How about: `Note: this feature is disabled by default, can be enabled by setting `rgw enable usage log = true` in the appropriate section of ceph.conf. For changes in ceph.conf to take effect, radosgw process restart is needed.`